app/settings.ini.example

```diff
@@ -0,0 +1,59 @@
+
+;;; base config
+
+[mpd]
+; if you set this to Rrue, the backend will connect to mpd (localhost:6600) and update
```

Comment:
; if you set this to True, the backend will connect to mpd (localhost:6600) and update